### ISSUE TITLE
Create Neural Network model for the proposed problem

### ISSUE THREAD

**Issue by allanbz** [allanbz/Arrhythmia-detector#5]
- [x] Define Network structure
- [x] Set Network parameters
- [x] Adjust Network to training data

**Comment by allanbz** [comment]
Using three Dense layers and a 20% Dropout layer.
